### ISSUE TITLE
Code Blocks in Visual Editor (VSCode): no input field for caption

### ISSUE THREAD

**Issue by joelvonrotz** [quarto-dev/quarto#320]
Hello,

I've noticed that there is no caption input field in the code block properties dialog and has to be added via Attributes:

![image](https://github.com/quarto-dev/quarto/assets/41155337/d889a0b8-82b8-4f29-b6f2-01a42efa8c8f)

Compared to for example the figures dialog.

![image](https://github.com/quarto-dev/quarto/assets/41155337/354c68ab-aeb7-442e-a36d-0ef135609e2a)

Of course, this doesn't mean _make code blocks dialog equal to the figure dialog_, as they're two different types of content. Just adding a separate input field would help a lot!

**Comment by jjallaire** [comment]
Indeed, we do need to take a fresh look at these dialogs to make sure they support all of the new standard attributes we've added in recent versions of Quarto. We will do this soon!

**Comment by joelvonrotz** [comment]
Thanks for your quick response! That makes a lot of sense.
Will there by chance also be some kind of dialog for tables?

**Comment by jjallaire** [comment]
Right now you can interact with tables primarily through the context or main tables menu. A dialog would indeed be a great improvement!
